docs/chaos/index.md

```diff
@@ -0,0 +1,53 @@
+# Chaos engineering with Simmy
+
+[Simmy][simmy] is a major new companion project adding a chaos-engineering and fault-injection dimension to Polly, through the provision of strategies to selectively inject faults, latency, custom behavior or fake results.
```

Comment:
```suggestion
[Simmy][simmy] is a major new addition to Polly library, adding a chaos-engineering and fault-injection dimension to Polly, through the provision of strategies to selectively inject faults, latency, custom behavior or fake results.
```
As @martincostello suggested, I would drop the `Project`, as it's now natively integrated into Polly.

Reply:
Done